internal/handlers/common/error.go

```diff
@@ -34,19 +34,46 @@ const (
 	// For ProtocolError only.
 	errInternalError = ErrorCode(1) // InternalError
 
-	ErrBadValue          = ErrorCode(2)     // BadValue
-	ErrFailedToParse     = ErrorCode(9)     // FailedToParse
-	ErrTypeMismatch      = ErrorCode(14)    // TypeMismatch
-	ErrNamespaceNotFound = ErrorCode(26)    // NamespaceNotFound
-	ErrNamespaceExists   = ErrorCode(48)    // NamespaceExists
-	ErrCommandNotFound   = ErrorCode(59)    // CommandNotFound
-	ErrNotImplemented    = ErrorCode(238)   // NotImplemented
-	ErrSortBadValue      = ErrorCode(15974) // Location15974
-	ErrInvalidArg        = ErrorCode(28667) // Location28667
-	ErrSliceFirstArg     = ErrorCode(28724) // Location28724
-	ErrProjectionInEx    = ErrorCode(31253) // Location31253
-	ErrProjectionExIn    = ErrorCode(31254) // Location31254
-	ErrRegexOptions      = ErrorCode(51075) // Location51075
+	// ErrBadValue for wrong input.
+	ErrBadValue = ErrorCode(2) // BadValue
+
+	// ErrFailedToParse failed to parse user input.
+	ErrFailedToParse = ErrorCode(9) // FailedToParse
+
+	// ErrTypeMismatch for $sort, when it is expected field sort to be of type object but is has other type.
```

Comment:
Are those codes some MongoDB's codes or something FerretDB-specific? The `it is expected field sort to be` part of the comment doesn't look clear to me. Is _"field sort"_ actually _field's sort_ or is it a field called _"sort"_? I was trying to find something Mongo-related for it in MongoDB docs, but I'm not sure where to look for it.